pkg/operator/sync.go

```diff
@@ -341,39 +347,90 @@ func (optr *Operator) syncMachineConfigController(config *renderConfig) error {
 		}
 	}
 
-	for _, path := range []string{
-		"manifests/machineconfigcontroller/events-rolebinding-default.yaml",
-		"manifests/machineconfigcontroller/events-rolebinding-target.yaml",
-	} {
+	for _, path := range paths.roleBindings {
```

Comment:
I'm wondering if syncMachineConfigPools() can't be pulled into your new func? isn't it checking manifests?

Reply:
It certainly could. I just didn't move that because it's a separate code path that I think is only taken once, and adding it would make the struct larger

Reply:
we can leave for now